### PR TITLE
Notes api

### PR DESCRIPTION
**Pull request by vijaytembugade**
- Added note creation.
- Archives
- Trash
- labels

### CODE REVIEW

File: src/Components/ArchivedNotes/ArchivedNotes.js [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
 
 const ArchivedNotes = () => {
-  return <div>ArchivedNotes</div>;
+  const {
+    archivesState: { archives },
+    archivesDispatch,
+  } = useArchives();
+
+  console.log(archives);
+
```
Comment (prathmeshjagtap) [inline src/Components/ArchivedNotes/ArchivedNotes.js:13]: you can remove the console.log

File: src/Components/LabeledNotes/LabledNotes.js [hunk 2]
```diff
@@ -1,7 +1,67 @@
...
+  const {
+    noteState: { notes },
+  } = useNotes();
+
+  const mainNotes = removeTrashedNotes(notes);
+  const labels = getLabels(mainNotes);
+
+  useEffect(() => {
+    console.log(notes);
+  });
```
Comment (prathmeshjagtap) [inline src/Components/LabeledNotes/LabledNotes.js:20]: you can remove this

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]

**Review by prathmeshjagtap — COMMENTED** [review]
Rest Everything is good and you have we'll structure your code, I liked that you have created a separate  function to get token from locally storage 

Just a Suggestion, Maybe raise small PR's from next time. of 500-600 lines max.

**Review by MarkVed17 — APPROVED** [review]
Functionality 💯 
Also the code looks clean.
Good to merge 👍🏼

**Pull request merged**